src/components/Breadcrumbs/index.tsx

```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
     component: any;
     hrefProperty: string;
   };
+  backgroundColor?: 'light' | 'dark';
   links: ILink[];
 }
 
+let background = 'light';
+const DarkMode =
+  'bg-gray-900  flex px-2 py-2 border rounded-lg  hover:border-gray-600';
 export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({
   links,
   customLink = {
     component: (props: any) => <a {...props} />,
     hrefProperty: 'href',
   },
+  backgroundColor = background,
 }) => {
-  return (
-    <nav className='flex' aria-label='Breadcrumb'>
-      <ol role='list' className='flex items-center space-x-4'>
-        <li>
-          <div>
-            <customLink.component
-              {...{ [customLink.hrefProperty]: links[0].href }}
-              className={classNames({
-                'text-gray-300': links.length > 1,
-                'hover:text-black': links.length > 1,
-                'hover:text-gray-200': links.length === 1,
-                'text-white': links.length === 1,
-              })}
-            >
-              <HomeIcon className='flex-shrink-0 w-5 h-5' aria-hidden='true' />
-              <span className='sr-only'>Home</span>
-            </customLink.component>
-          </div>
-        </li>
-        {links.slice(1).map((page, i) => (
-          <li key={i} title={page.title}>
-            <div className='flex items-center'>
-              <ChevronRightIcon
-                className='flex-shrink-0 w-5 h-5 text-gray-300'
-                aria-hidden='true'
-              />
+  if (backgroundColor === 'light') {
```

Comment:
instead of doing this if statement, just do ternary for the classnames checking for darkMode